builtin/credential/ldap/path_config.go

```diff
@@ -104,6 +124,9 @@ func (b *backend) Config(req *logical.Request) (*ConfigEntry, error) {
 	if err := entry.DecodeJSON(&result); err != nil {
 		return nil, err
 	}
+
+	// Make logger available for later
+	result.Logger = b.Logger()
```

Comment:
As mentioned later, just make the other functions members of `b` and use `Logger()` directly.